### ISSUE TITLE
[rllib] How does rllib sync torch model buffers in distributed learning?

### ISSUE THREAD

**Issue by Purpleslz** [ray-project/ray#11967]
In https://github.com/ray-project/ray/blob/master/rllib/policy/torch_policy.py#L337 
```
    def learn_on_batch(
            self, postprocessed_batch: SampleBatch) -> Dict[str, TensorType]:
        # Compute gradients (will calculate all losses and `backward()`
        # them to get the grads).
        grads, fetches = self.compute_gradients(postprocessed_batch)

        # Step the optimizers.
        for i, opt in enumerate(self._optimizers):
            opt.step()

        if self.model:
            fetches["model"] = self.model.metrics()
        return fetches
```
It only cares parameters with gradients, how does rllib sync model buffers(normalization parameters, which does not have grads) ?

**Comment by michaelzhiluo** [comment]
For layers such as Batchnorm, I believe the running mean and std are initialized independently for each worker.

There is an example of normalization sync [here](https://github.com/ray-project/ray/blob/master/rllib/agents/mbmpo/mbmpo.py#L286) if you want to try it on your own.

**Comment by Purpleslz** [comment]
> For layers such as Batchnorm, I believe the running mean and std are initialized independently for each worker.
> 
> There is an example of normalization sync [here](https://github.com/ray-project/ray/blob/master/rllib/agents/mbmpo/mbmpo.py#L286) if you want to try it on your own.

I think the better solution is sync buffers during training phase in policy. In this way, rllib could sync normalization parameters across all training algorithms.
